proto/skaffold.proto

```diff
@@ -48,6 +50,15 @@ message BuildMetadata {
   map<string, string> additional = 99;
 }
 
+message TestMetadata {
+    message Tester {
```

Comment:
Please add description to all newly added structs and field  to make them appear in skaffold.dev docs.

Reply:
Please add label `docs-modifications` to the PR to generate docs skaffold.dev and verify if nested structs render correctly as expected.